Sources/Stencil/AssetsCatalogContext.swift

```diff
@@ -38,11 +38,19 @@ extension AssetsCatalogParser {
       switch entry {
       case let .group(name: name, items: items):
         return [
+          "type": "group",
           "name": name,
           "items": structure(entries: items)
         ]
+      case let .color(name: name, value: value):
+        return [
+          "type": "color",
+          "name": name,
+          "value": value
```

Comment:
The "value" used here is not the "color value" as in "RGB components". That name could be misleading.
I suggest we structure that context with `"type"="color"`, `"name"="basename-of-the-color-without-group-path"`, `"fullname"="name-of-the-color-with-group-path-to-use-in-init"` but also `red`, `green` `blue` and `alpha` keys to match what's output by the `ColorParser`, in case some people want to use that for anything other than `UIColor(name: "full name")` — e.g. for generating some some HTML to display the colors or whatnot…